src/Microsoft.ML.LightGBM/Microsoft.ML.LightGBM.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeInPackage>Microsoft.ML</IncludeInPackage>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.ML.Core\Microsoft.ML.Core.csproj" />
+    <ProjectReference Include="..\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
+    <ProjectReference Include="..\Microsoft.ML.FastTree\Microsoft.ML.FastTree.csproj" />
+    <ProjectReference Include="..\Microsoft.ML.StandardLearners\Microsoft.ML.StandardLearners.csproj" />
+    <ProjectReference Include="..\Microsoft.ML\Microsoft.ML.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Properties\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="LightGBM" Version="2.1.2.2" />
```

Comment:
You’ll need to add a new nupkg project for this. Not all users will need/want LightGBM, so we shouldn’t require everyone to depend on/deploy it.

Reply:
Can you please clarify? My understanding is LightGBM will always be part of ML.NET

---
In reply to: [197500282](https://github.com/dotnet/machinelearning/pull/392#discussion_r197500282) [](ancestors = 197500282)

Reply:
Part of ML.NET, but in a separate nuget so those that don't need it don't get it. There would be the ML.NET nuget, and the ML.NET LightGBM nuget, the ML.NET OnnxExport nuget, the ML.NET FastTree nuget, the ML.NET CNTK nuget, the ML.NET torch nuget, etc.

---
In reply to: [197566747](https://github.com/dotnet/machinelearning/pull/392#discussion_r197566747) [](ancestors = 197566747,197500282)